packages/component-library/draft/Kaizen/ZenOffCanvas/ZenOffCanvas.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from "react"
 
-import { ColorScheme } from "@kaizen/component-library/draft/Kaizen/ZenNavigationBar"
+import { ColorScheme } from "@kaizen/draft-zen-navigation-bar/KaizenDraft/ZenNavigationBar/NavigationBar"
```

Comment:
This was failing for some reason, even though it's a (old style) draft, and shouldn't need to point to the new drafts. I've only done this to satisfy the build